README.adoc

```diff
@@ -97,27 +97,23 @@ include::{common-includes}/skills-network-kube-setup.adoc[]
 
 Navigate to `start` directory to begin.
 
-The `inventory` microservice should be healthy only when `system` is available. To add this 
-check to the `/health/ready` endpoint, you will create a class that is annotated with the
-`@Readiness` annotation and implements the `HealthCheck` interface.
-
 [role="code_command hotspot", subs="quotes"] 
 ---- 
-#Create the `InventoryReadinessCheck` class.#
-`inventory/src/main/java/io/openliberty/guides/inventory/InventoryReadinessCheck.java`
+#Create the `InventoryStartupCheck` class.#
+`inventory/src/main/java/io/openliberty/guides/inventory/InventoryStartupCheck.java`
 ---- 
 
-InventoryReadinessCheck.java
+InventoryStartupCheck.java
 [source, Java, linenums, role='code_column hide_tags=copyright']
 ----
-include::finish/inventory/src/main/java/io/openliberty/guides/inventory/InventoryReadinessCheck.java[]
+include::finish/inventory/src/main/java/io/openliberty/guides/inventory/InventoryStartupCheck.java[]
 ----
 
-This health check verifies that the `system` microservice is available at 
-`\http://system-service:9080/`. The `system-service` host name is only accessible from 
-inside the cluster, you can't access it yourself. If it's available, then it returns an 
-`UP` status. Similarly, if it's unavailable then it returns a `DOWN` status. When the 
-status is `DOWN`, the microservice is considered to be unhealthy.
+A health check for startup allows applications to define startup probes that are used for initial verification of the application before the liveness probe takes over. 
+This is useful for applications which require additional startup time on their first initialization.
+The `@Startup` annotation must be applied on a HealthCheck implementation to define a startup check procedure, otherwise, this annotation is ignored.
```

Comment:
The `@Startup` annotation must be applied on a HealthCheck implementation to define a startup check procedure, otherwise, this annotation is ignored.
--->
The `@Startup` annotation must be applied on a HealthCheck implementation to define a startup check procedure. Otherwise, this annotation is ignored.